src/platforms/components/FarcasterIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import { Icon } from "@chakra-ui/react"
+import { IconProps } from "phosphor-react"
+import { ForwardedRef, forwardRef } from "react"
+
+function FarcasterIcon(_: IconProps, ref: ForwardedRef<SVGSVGElement>) {
```

Comment:
I wonder if we could just put this icon inside the `static` folder and import it just like we import e.g. the user avatar icons?

Reply:
Great idea! Change: https://github.com/guildxyz/guild.xyz/pull/1301/commits/a12328008b154470df1916fe2b61aabfc0d0719c